src/components/Class/index.js

```diff
@@ -39,27 +41,33 @@ function Class({ classToEdit }) {
     end_time,
     course_id,
     pathway_id,
+    exercise_id,
+    max_enrolment,
   } = classToEdit;
 
-  const initialFormState = useRef({
-    [TITLE]: title || "",
-    [DESCRIPTION]: description || "",
-    [FACILITATOR_NAME]: facilitator ? facilitator.name : "",
-    [FACILITATOR_EMAIL]: facilitator ? facilitator.email : "",
-    [START_TIME]: start_time
-      ? moment.utc(start_time).format("YYYY-MM-DD")
-      : moment().format("YYYY-MM-DD"),
-    [CLASS_START_TIME]: start_time
-      ? moment.utc(start_time).format("kk:mm")
-      : moment().format("kk:mm"),
-    [CLASS_END_TIME]: end_time
-      ? moment.utc(end_time).format("kk:mm")
-      : moment().add(15, "minute").format("kk:mm"),
-    [LANG]: lang || "hi",
-    [TYPE]: type || "doubt_class",
-    [COURSE_ID]: course_id || "",
-    [PATHWAY_ID]: pathway_id || "",
-  });
+  const initialFormState = useMemo(() => {
+    return {
+      [TITLE]: title || "",
```

Comment:
Just wanted to pick your brain about this.
I haven't encountered this syntax earlier, is it some pattern or new feature ?

Thanks.